### DISCUSSION TITLE
Is the slider picker supposed to be interactable?

### DISCUSSION THREAD

**Discussion by frutejuise** [z-huang/music#1427] [Q&A] [answered]
I can change where the sliders are and I'm not sure that's supposed to happen.

![Screenshot_20240821-072005.png](https://github.com/user-attachments/assets/ae122af8-f9dc-4d97-badf-67a0d88d379c)

![Screenshot_20240821-072002.png](https://github.com/user-attachments/assets/0d69fd2a-eefb-4a56-889c-0e31a2aef355)

**Comment by z-huang** [comment, marked as answer]
It says player slider style, so swipe up the bottom player sheet and you'll see the difference. The sliders are designed to be interactable to let user play and better decide which one they want.
